tools/PI/DevHome.PI/Controls/ExternalToolsManagementButton.cs

```diff
@@ -15,7 +15,7 @@ namespace DevHome.PI.Controls;
 
 internal sealed class ExternalToolsManagementButton : Button
 {
-    private const string _UnregisterButtonText = "\uECC9";
+    private const string UnregisterButtonText = "\uECC9";
```

Comment:
NIT: This is a glyph, not text.  Can the name be changed?

Reply:
Sure.